### ISSUE TITLE
PIL/Pillow problem on Python 3

### ISSUE THREAD

**Issue by SectOLT** [Nandaka/PixivUtil2#610]
### Prerequisites

* [x] Did you read FAQ section?
* [X] Did you test with the latest [releases](https://github.com/Nandaka/PixivUtil2/releases "latest releases") or [commit](https://github.com/Nandaka/PixivUtil2/commits/master "latest commit") ?

### Description

I have had problems since the PixivUtil switched from Python 2.7 to Python 3.8, and now this has gotten a bit more problematic than before. I am now experiencing errors with PIL and Pillow, while Pillow is prefered since PIL is deprecated, the tool requests that i install PIL for good working, however, upon doing so, PIL returns errors to which i tried different methods, cleaning, reinstalling, of each PIL and Pillow, but to no avail, i still come across the same problem, which is: 

ImportError: cannot import name '_imaging' from 'PIL' (/usr/lib/python3/dist-packages/PIL/__init__.py)

[pixivutil.log](https://github.com/Nandaka/PixivUtil2/files/4013724/pixivutil.log)

I have no ideas what to do at this point, it seems as if whatever i attempt to do, i still get this problem, and if i do attempt to uninstall PIL as well as Pillow then install Pillow only, i get told that that PIL module cannot be found and therefore the PixivUtil does not work in the first place.

### Steps to Reproduce

1. Run python PixivUtil2.py
2. Try to download an image
3. Wait

**Expected behavior:** The image should have downloaded as intended

**Actual behavior:** Error returned, described above

### Versions

v20191221

**Comment by Nandaka** [comment]
weird, I don't have PIL anymore in my package list
```
Package             Version
------------------- ----------
apng                0.3.3
astroid             2.3.3
atomicwrites        1.3.0
attrs               19.3.0
beautifulsoup4      4.8.1
cachetools          4.0.0
certifi             2019.11.28
chardet             3.0.4
colorama            0.4.3
demjson             2.2.4
entrypoints         0.3
flake8              3.7.9
future              0.18.2
html5lib            1.0.1
idna                2.8
imageio             2.6.1
isort               4.3.21
lazy-object-proxy   1.4.3
mccabe              0.6.1
mechanize           0.4.4
more-itertools      8.0.2
numpy               1.17.4
packaging           19.2
pefile              2019.4.18
Pillow              6.2.1
pip                 19.3.1
pluggy              0.13.1
py                  1.8.0
py2exe              0.9.4.0
pycodestyle         2.5.0
pyflakes            2.1.1
pylint              2.4.4
pyparsing           2.4.5
pytest              5.3.2
requests            2.22.0
setuptools          41.2.0
six                 1.13.0
SocksiPy-branch     1.1
soupsieve           1.9.5
urllib3             1.25.7
wcwidth             0.1.7
webencodings        0.5.1
wheel               0.33.6
win-unicode-console 0.5
wrapt               1.11.2
```

**Comment by SectOLT** [comment]
Yeah i know that, which is why i am confused, if PixivUtil2 does not rely on PIL to work, why does it says PIL module not found in the first place when i do try to run PixivUtil2, so i am stuck here

**Comment by SectOLT** [comment]
Okay, here's the problem i have found, in theory, it's because of it that things are breaking.

PIL short for Pillow, while PIL is old and deprecated, Pillow is a fork of it that is still being maintained.
Now what happens here, is that my environment is Cinnamon, this relies on PIL to work, no matter what, i will always be forced to reinstall python-pil using the software manager because when i try to install python-pillow, it defaults back to python-pil instead. the other problem is that if i try to install pillow using pip, the link will not be created, while Pillow contains the libs that PIL offers, PixivUtil2 fails to find those links and tnerefore cannot access the libs offered by Pillow

I am not sure how to fix that, but if i want this to work, i would need to teach PixivUtil that Pillow is there somewhere on the system and use it, because by default, it tries to use my system's current python-pil package

**Comment by Nandaka** [comment]
I'm not sure, but usually I install the required packages using pip bundled with python following the instruction: https://pillow.readthedocs.io/en/stable/installation.html#linux-installation

other option is to disable the image verification setting in config.ini by setting `verifyImage = False`

**Comment by SectOLT** [comment]
I see, so chances are that this issue has been there for me this entire time, just that it never got brought up because i never enabled verification until now
